indico/modules/events/surveys/templates/management/survey_submission.html

```diff
@@ -21,13 +21,26 @@
 {% endblock %}
 
 {% block content %}
-    <div class="i-box-content">
-        <dl class="i-data-list">
-            {% for answer in submission.answers -%}
-                <dt><strong>{{ answer.question.title }}</strong></dt>
-                <dd>{{ _render_answer(answer) }}</dd>
-            {%- endfor %}
-        </dl>
+    <div id="survey-submission" class="i-box-content">
+        {% for section in survey.sections if section.children -%}
```

Comment:
I think the nested loop here will send multiple SQL queries (one for each section to get its children).

Adding this in `RHSurveySubmissionBase` would probably avoid this:

    sections_strategy = defaultload('sections').joinedload('children')

And in the query options:

    .options(answers_strategy, survey_strategy, sections_strategy)